opstack/src/builder.rs

```diff
@@ -92,7 +92,16 @@ impl OpStackClientBuilder {
         );
         let consensus = ConsensusClient::new(&config);
         let fork_schedule = ForkSchedule {
-            prague_timestamp: u64::MAX,
+            bedrock_timestamp: 1686068903,
+            regolith_timestamp: 0,
```

Comment:
Why is regolith enabled at timestamp 0? I think it occurred sometime around bedrock.